frontend/packages/dev-console/integration-tests/views/git-import-flow.view.ts

```diff
@@ -20,7 +20,9 @@ export const builderImage = element(
   by.cssContainingText('.pf-c-card.odc-builder-image-card', 'Node.js'),
 );
 export const buildImageVersion = element(by.id('form-dropdown-image-tag-field'));
-export const createButton = element(by.css('[data-test-id="import-git-create-button"]'));
+export const createButton = element(by.css('[data-test-id="import-git-form"]')).element(
+  by.css('Button[data-test-id="submit-button"]'),
```

Comment:
You're not adding `Button` to the other selectors... any reason why this one?

Reply:
No particular reason to add Button, as we have data test ids as primary selector removing Button should not be an issue. removed it.